packages/component/src/Activity/StackedLayout.tsx

```diff
@@ -159,9 +161,7 @@ const StackedLayout: FC<StackedLayoutProps> = ({
       })}
     >
       <div className="webchat__stacked-layout__main">
-        <div className="webchat__stacked-layout__avatar-gutter">
-          {showAvatar && renderAvatar && renderAvatar({ activity })}
-        </div>
+        <div className="webchat__stacked-layout__avatar-gutter">{showAvatar && renderAvatar()}</div>
```

Comment:
In line 146, `const showAvatar = showCallout && hasAvatar && !!renderAvatar`, it implies "a truthy `showAvatar` means `renderAvatar` is also truthy".

So we don't need to check truthiness of `renderAvatar` here.